src/h3lib/include/h3api.h.in

```diff
@@ -385,40 +385,6 @@ H3Error H3_EXPORT(exactEdgeLengthM)(H3Index edge, double *length);
  *
  * It works out to be `2 + 120*7^r` for resolution `r`.
  *
- * # Mathematical notes
```

Comment:
I don't mind having this here as well as in the docs - have referred to it several times to get the formula for the count of pentagon children. Doesn't seem like there's a cost to leave it here, as it's not going to change - at very least, a link to the docs page that discusses the math would be nice.

Reply:
OK. I was worried it was cluttering the `.h` file, but happy to leave it if it's been helpful!